### PR TITLE
MBS-9833: Link relationship style guide within release edit relationships tool

### PR DESCRIPTION
**Pull request by y-young**
# Problem

[MBS-9833](https://tickets.metabrainz.org/browse/MBS-9833): Link relationship style guide within release edit relationships tool

# Solution

Add relationship style guide link to edit relationships tool 

# Checklist for author

* [x] Read the [Contributing Guidelines](https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md)

# Action

1. Should we update the corresponding .po file for translation?
2. I'm sure if this message is the best choice, so please let me know if it can be better :)

### CODE REVIEW

File: root/release/edit_relationships.tt [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
       <p>[% l('To use the batch tools, select some recordings or works using the checkboxes.') %]</p>
 
+      <p>[% l('For more information, see {csg_relationships|relationship style guideline}.', { csg_relationships => { href => doc_link('Style/Relationships'), target => '_blank' } }) %]</p>
```
Comment (reosarevok) [inline root/release/edit_relationships.tt:20]:
```suggestion
      <p>[% l('Please read {relationships_doc|our guidelines for relationships} if you haven’t already.', { relationships_doc => { href => doc_link('Style/Relationships'), target => '_blank' } }) %]</p>
```

csg => classical style guide, it probably applied wherever you copied this from but not here :) 
Changed the text a bit to make it more clear that we expect people to follow those guidelines - let me know what you think :)

**Review by reosarevok — CHANGES_REQUESTED** [review]

**Comment by reosarevok** [comment]
@brainzbot, add to whitelist

**Comment by reosarevok** [comment]
The CircleCI test failure is spurious, ignore it :) Also, do you have a JIRA account? (you can log in with MusicBrainz nowadays, if not). Please add yourself as the assignee of MBS-9833.

**Comment by y-young** [comment]
> The CircleCI test failure is spurious, ignore it :) Also, do you have a JIRA account? (you can log in with MusicBrainz nowadays, if not). Please add yourself as the assignee of [MBS-9833](https://tickets.metabrainz.org/browse/MBS-9833).
> 
> The .po files get updated by the team once we merge this while we put the code in beta, so you don't need to worry about that :)

You're right, I didn't understand what happened when looking at the error message.

All done, thanks for your help!

**Review by reosarevok — APPROVED** [review]

**Review by yvanzo — APPROVED** [review]
Thank you (again) @y-young! I will merge without squashing because commit messages look nice. :+1:

**Pull request merged**